resources/geocoding/he/972.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2017 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Sourced from https://he.wikipedia.org/wiki/קידומת_טלפון_בישראל
+
+9722|ירושלים
+9723|תל אביב-יפו והמרכז
+9724|חיפה והצפון
+9728|השפלה והדרום
+9729|השרון
+
```

Comment:
Please remove extra line

(I see we also have an extra line at the end of the English file for 972.txt, I'll remove that too unless you want to do that in this PR).